src/vue/modules/coinpayments-deposit/components/pending-issuances-table-row.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <tbody class="deposit-table-row">
```

Comment:
the file named pending-issuances-table-row
but the block named deposit-table-row

Reply:
ok